Mathlib/Topology/Algebra/Module/Alternating/Topology.lean

```diff
@@ -138,6 +148,10 @@ theorem hasBasis_nhds_zero :
 
 variable [ContinuousSMul 𝕜 E]
 
+lemma closedEmbedding_toContinuousMultilinearMap [T2Space F] :
+    ClosedEmbedding (toContinuousMultilinearMap : (E [⋀^ι]→L[𝕜] F → _)) :=
```

Comment:
Please spell out the meaning of `_`, otherwise non-experts of this file may have a difficulty parsing the statement. Also, add parentheses like above?